src/__tests__/3box.test.js

```diff
@@ -114,7 +125,8 @@ describe('3Box', () => {
     const IPFS_OPTIONS = {
       EXPERIMENTAL: {
         pubsub: true
-      }
+      },
+      repo: './.js-ipfs'
```

Comment:
Why this path rather than the ./tmp/ipfs<n> that we have been using so far?

Reply:
testsweren't using ' ./tmp/ipfs' at the moment, maybe that was lost somewhere else in another change, can check, but yeah lets use tmp